SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -770,7 +770,8 @@ int64 USpatialActorChannel::ReplicateActor()
 				const VirtualWorkerId NewAuthVirtualWorkerId = NetDriver->LoadBalanceStrategy->WhoShouldHaveAuthority(*Actor);
 				if (NewAuthVirtualWorkerId == SpatialConstants::INVALID_VIRTUAL_WORKER_ID)
 				{
-					UE_LOG(LogSpatialActorChannel, Error, TEXT("Load Balancing Strategy returned invalid virtual worker for actor %s"), *Actor->GetName());
+					const FVector SpatialPosition = SpatialGDK::GetActorSpatialPosition(Actor);
+					UE_LOG(LogSpatialActorChannel, Error, TEXT("Load Balancing Strategy returned invalid virtual worker for actor %s (Spatial position: %s)"), *GetNameSafe(Actor), *SpatialPosition.ToString());
```

Comment:
Can you move this error to the end of GridBaseLBStrategy::WhoShouldHaveAuthority? We already access the spatial position and bonus we're only printing the position for the spatially relevant load balancing strategies.

Reply:
Done!  Preserved original error so we will still catch this failure coming from non-GridBased lb strategies.